src/ol_infrastructure/applications/mitxpro/__main__.py

```diff
@@ -94,11 +95,8 @@
     description="Access from the mitxpro and operations VPC to the mitxpro edxapp database",
     ingress=[
         ec2.SecurityGroupIngressArgs(
-            security_groups=[
-                mitxpro_edxapp_security_group.id,
-                mitxpro_edx_worker_security_group.id,
```

Comment:
We can maintain the security groups block, we just want to add the operations CIDR block for allowing traffic from Vault.